README.md

```diff
@@ -197,6 +197,10 @@ Here is how to run Eclair with plugins:
 eclair-node-<version>-<commit_id>/bin/eclair-node.sh <plugin1.jar> <plugin2.jar> <...>
 ```
 
+### List of current usable plugins
+
+- https://github.com/engenegr/eclair-alarmbot-plugin
+
```

Comment:
We must make it clear that these plugins are not developed by Acinq (we don't have the bandwidth to do support for these plugins). What about:

```suggestion
### Non-exhaustive plugins list

Here are some plugins created by the eclair community.
If you need support for these plugins, head over to their respective github repository.

- [Telegram Bot for eclair alerts](https://github.com/engenegr/eclair-alarmbot-plugin)

```